pkg/client/actionclient.go

```diff
@@ -99,13 +102,23 @@ func AppendInstallFailureUninstallOptions(opts ...UninstallOption) ActionClientG
 		return nil
 	}
 }
+
 func AppendUpgradeFailureRollbackOptions(opts ...RollbackOption) ActionClientGetterOption {
 	return func(getter *actionClientGetter) error {
 		getter.upgradeFailureRollbackOpts = append(getter.upgradeFailureRollbackOpts, opts...)
 		return nil
 	}
 }
 
+type PostRendererProvider func(rm meta.RESTMapper, kubeClient kube.Interface, obj client.Object) postrender.PostRenderer
+
+func AppendPostRenderers(postRendererFns ...PostRendererProvider) ActionClientGetterOption {
```

Comment:
@ludydoo please correct me if I'm wrong. Could we add a test to verify if `postRendererProvider` is added to `chainedPostRenderer` and is run during install. A mock postrenderer could be as simple as adding an additional annotation to an object or maybe need not even interact with the cluster and use k8s client. Thoughts?

Reply:
done!